Tests iOS/MyList/SavesTests.swift

```diff
@@ -312,10 +312,17 @@ class SavesTests: XCTestCase {
         XCTAssertEqual(listView.skeletonCellCount, 4)
         promises[0].completeWith(.success(Response.saves("saves-loading-page-1")))
 
+        if listView.itemCount != 2 {
+//            This conditional tries fix the flakey test failures by reloading saves
```

Comment:
seems like we're doing a check to see if the itemCount equals, otherwise do another reload. What if it fails again? Do we know why it fails the first time?

Reply:
I can't say for sure, but I was able to replicate a failure every time in Xcode by setting a breakpoint after the item load, and then removing the breakpoint and re-running - it would fail after the removal. This fix solved the problem, but I have no way of knowing for sure if Bitrise fails for the same reason so if this doesn't work I'll have to try something else.

Reply:
thanks for looking into this! approving as this will help avoid the test from failing